prescripted_countries/btr_prescripted_countries.txt

```diff
@@ -1288,23 +1288,22 @@ parnack_regime={
     species_adjective = "PRESCRIPTED_species_adjective_parnack_regime"
     #species_bio="PRESCRIPTED_species_bio_parnack_regime"	
 
-	authority="auth_dictatorial"
-	government="gov_military_dictatorship"
-	civics={ "civic_warrior_culture" "civic_barbaric_despoilers" }
+	authority = "auth_oligarchic"
+	government= "gov_military_junta"
+	civics={ "btr_civic_pack_ment" "btr_civic_enfor_iso" }
 
 	flags={ custom_start_screen parnack_regime btr_empire canon_map hybrid_map sandbox_map btr_minor parnack_regime_player }
 
-	ethic = "ethic_fanatic_xenophobe"
+	ethic = "ethic_xenophobe"
 	ethic = "ethic_militarist"
+	ethic = "ethic_authoritarian"
 
-	trait="btr_yahg_physique"
-	trait="trait_deviants"
-	trait="trait_quarrelsome"
-	trait="btr_yahg_keen_mind"
-	trait="trait_repugnant"
-	trait = "trait_yahg_habitability_preference"

```

Comment:
Add this back in so Puck doesn't kill us.

Reply:
trait = "trait_yahg_habitability_preference" specifically this one

Reply:
Oh yea good point!